packages/runtime/container-runtime/src/scheduleManager.ts

```diff
@@ -0,0 +1,310 @@
+import EventEmitter from "events";
```

Comment:
Can you please make this move as a separate PR? I.e. separate mechanical changes (like move) from the rest of the changes? It will make it easier to see what actually changed in this code, if anything

Reply:
Yeah let me move this back so that the PR is easier to look through. I forgot I had moved it until I started opening the draft.